packages/PyTrilinos/src/CMakeLists.txt

```diff
@@ -60,9 +60,9 @@ IF(PyTrilinos_ENABLE_STK)
   SET(HAVE_STK ON)
 ENDIF()
 
-IF(NOX_ENABLE_Epetra)
+IF(NOX_ENABLE_ABSTRACT_IMPLEMENTATION_EPETRA)
```

Comment:
If NOX is disabled, this CMake option is not defined.  You could do this:
```
IF(DEFINED NOX_ENABLE_ABSTRACT_IMPLEMENTATION_EPETRA AND NOX_ENABLE_ABSTRACT_IMPLEMENTATIN_EPETRA)
```
or define a CMake option in PyTrilinos for this: `PyTrilinos_NOX_ENABLE_ABSTRACT_IMPLEMENTATION_EPETRA`.